### PR TITLE
Display derivation path for HD watch-only wallets

### PR DESCRIPTION
**Pull request by demerzel3**
Closes https://github.com/BlueWallet/BlueWallet/issues/3371

### CODE REVIEW

**Review by Overtorment — APPROVED** [review]
lgtm

File: tests/unit/fixtures/skeleton-cobo.txt [hunk 1]
```diff
@@ -1 +1 @@
-{"ExtPubKey":"zpub6rcabYFcdr41zyUNRWRyHYs2Sm86E5XV8RjjRzTFYsiCngteeZnkwaF2xuhjmM6kpHjuNpFW42BMhzPmFwXt48e1FhddMB7xidZzN4SF24K","MasterFingerprint":"5271c071","CoboVaultFirmwareVersion":"1.2.4(BTC-Only)"}
\ No newline at end of file
+{"ExtPubKey":"zpub6rcabYFcdr41zyUNRWRyHYs2Sm86E5XV8RjjRzTFYsiCngteeZnkwaF2xuhjmM6kpHjuNpFW42BMhzPmFwXt48e1FhddMB7xidZzN4SF24K","MasterFingerprint":"5271c071","AccountKeyPath":"84'\/0'\/0'","CoboVaultFirmwareVersion":"1.2.4(BTC-Only)"}
```
Comment (limpbrains) [inline tests/unit/fixtures/skeleton-cobo.txt:1]: Is this an actual skeleton file from Cobo? 
If this file doesn't usually contains `AccountKeyPath`, maybe we should keep it this way in tests?

Comment (Overtorment) [reply tests/unit/fixtures/skeleton-cobo.txt:1]: i just doublechecked my cobo device, and it has `AccountKeyPath`. i dunno why I didn't include it in this fixture. maybe on old firmware `AccountKeyPath` was not exported...

**Review by limpbrains — APPROVED** [review]

**Review by marcosrdz — APPROVED** [review]

**Comment by Overtorment** [comment]
thanks!

**Pull request merged**